devito/ir/equations/equation.py

```diff
@@ -157,7 +167,7 @@ def __new__(cls, *args, **kwargs):
 
         expr._dspace = dspace
         expr._ispace = ispace
-        expr._conditionals = frozendict([(d, ()) for d in conditionals])
+        expr._conditionals = conditionals
```

Comment:
Isn't that the revert of yesterday? 🤣

Reply:
the isolated line yeah... but in practice no! [these lines](https://github.com/devitocodes/devito/pull/1421/files#diff-14cbfe70a6d744024416f98cb417dcbaR153-R161) :grimacing: